radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/HighQC.java

```diff
@@ -159,7 +161,7 @@ public static HighQC from(
 		QuorumCertificate highestCommittedQC,
 		Optional<TimeoutCertificate> highestTC
 	) {
-		return new HighQC(highestQC, Objects.requireNonNull(highestCommittedQC), highestTC.orElse(null));
+		return new HighQC(highestQC, requireNonNull(highestCommittedQC), highestTC.orElse(null));
```

Comment:
Constructor accepts null `highestCommittedQC`, but here we require it to be non null.

Reply:
done